test/extended/util/annotate/rules.go

```diff
@@ -94,6 +94,14 @@ var (
 			`\[sig-cli\] Kubectl Port forwarding With a server listening on localhost should support forwarding over websockets`,
 			`\[sig-cli\] Kubectl Port forwarding With a server listening on 0.0.0.0 should support forwarding over websockets`,
 			`\[sig-node\] Pods should support remote command execution over websockets`,
+
+			// These tests are flacky and require internet access
+			// See https://bugzilla.redhat.com/show_bug.cgi?id=2019375
```

Comment:
can you make a note here listing the bugs that are being used to investigate the failures (i.e. the bugs that, when resolved, will allow us to either re-enable these tests, or decide to skip them permanently)?

would be good to have that note in master as well.  This is how someone 6 months from now who is digging through this, can follow up on what ended up happening and ensure the tests are properly re-enabled(if they should be).

Reply:
@bparees done via https://github.com/openshift/origin/pull/26710 - do you want me to add it here as well?

Reply:
being done in master here:
https://github.com/openshift/origin/pull/26710

less critical to have it in the z-branch, since it'll eventually carry backwards from master over time and we're unlikely to backport the fixes to re-enable these tests in 4.9.